javascriptv3/example_code/cross-services/transcription-app/setup.yaml

```diff
@@ -0,0 +1,330 @@
+Resources:
```

Comment:
Minor comment, but we've talked about possibly adding a comment to our CloudFormation .yaml files about where the backing CDK source code is, for those who want to play with it. What do you think?

Reply:
Implemented

Reply:
Broken URL to GitHub?

Reply:
fixed